node/Cargo.toml

```diff
@@ -30,7 +30,7 @@ module-amm-rpc = { path = '../pallets/amm/rpc', version = '2.0.0' }
 
 # Substrate dependencies
 frame-benchmarking = '2.0.0'
-frame-benchmarking-cli = '2.0.0'
+frame-benchmarking-cli = {version = '2.0.0', optional = true }
```

Comment:
Is this formatted?

Reply:
done. looks better, consistent now.

Reply:
Was it actually formatted or did you fix this manually? 

Maybe we just need to set some settings. I've noticed similar stuff in the code a few times.

Reply:
I did it manually. Is there something which can format these type of files ? i guess there is but apparently i dont use it yet.